docs/release-maintenance.md

```diff
@@ -0,0 +1,87 @@
+# gluster-kubernetes Release and Maintenance Policies
+
+This document outlines the release and maintenance policies of the
+gluster-kubernetes project.
+
+## Maintained Branches
+
+The project will only support and actively maintain two branches, `master` and
+`stable`. Requests for support of older branches may be considered on a
+case-by-case basis, but users will be encouraged to use newer versions where
+possible.
+
+## Version Numbering
+
+This project follows the versioning guidelines outlined by the [Semantic
+Versioning specification, version 2.0.0](http://semver.org/spec/v2.0.0.html).
+In short, versions numbers will follow the structure `<MAJOR>.<MINOR>.<PATCH>`,
+with the following definitions:
+
+ * MAJOR version indicates a fundamental change to the structure of the
+   project, often due to innovations from significant changes in the component
+   projects. Major versions will typically not be compatible with older
+   versions of the component projects.
```

Comment:
Just to understand, this means, e.g. when we introduced a new way of deploying gluster, e.g. with the DaemonSet, this would indicate a MAJOR version change (if we had done this versioning at the time) since previous versions of kubernetes supported in the earlier version of gluster-kubernetes, did not support DaemonSet -- correct?